packages/dds/tree/src/simple-tree/node-kinds/object/objectNode.ts

```diff
@@ -495,10 +454,7 @@ export function objectSchema<
 			instance: TreeNodeValid<T2>,
 			input: T2,
 		): UnhydratedFlexTreeNode {
-			return UnhydratedFlexTreeNode.getOrCreate(
-				unhydratedContext,
-				mapTreeFromNodeData(input as object, this as unknown as ImplicitAllowedTypes),
-			);
+			return mapTreeFromNodeData(input as object, this as unknown as ImplicitAllowedTypes);
```

Comment:
Nit: a quick comment to explain the cast through `unknown` would probably be good

Reply:
Tweaked to avoid casting through unknown